pkg/image/apis/image/helper.go

```diff
@@ -41,6 +46,16 @@ const (
 	ImportRegistryNotAllowed = "registry is not allowed for import"
 )
 
+// knownSignatureStores is a list of known signature stores. The key represents
+// a registry name and it maps to a signature store for that registry. The
+// signature store has to be HTTPS and it must offer image signatures based on
+// this format: https://STORE_URL/IMAGE_NAME/signature-1
+// The response must be valid image signature blob.
+// TODO: Maybe this should be configurable via master config?
```

Comment:
So our options appear to be

1. master reads `/etc/containers/registries.d/*` files. This would be introducing a special and new OpenShift config pattern.
1. add to `master-config.yaml`.

With option 2 we could at least align with the [sigstore config defined in containers/image](https://github.com/containers/image/blob/master/docs/registries.d.md#using-containers-from-various-origins), something like:

```yaml
imagePolicyConfig:
  registrySignaturesForImport:
  -  registry.access.redhat.com:
        sigstore: https://access.redhat.com/webassets/docker/content/sigstore
```